app/validations/address.js

```diff
@@ -36,5 +36,13 @@ export function getAddressValidations(isAlwaysRequired = false) {
         ? null
         : ['street', 'number', 'postcode', 'province'],
     }),
+    province: validatePresence({
```

Comment:
I think this will make it required again for the position pages as well?

Reply:
waiting for reply of sofie